src/components/custom/ZwaveGraph.vue

```diff
@@ -1196,6 +1204,7 @@ export default {
 					entity,
 					node.statistics?.nlwr,
 					RouteKind.NLWR,
+					!!node.statistics?.lwr,
```

Comment:
@kpine in fact, I have added a duplicated `!` here, that's a typo

```suggestion
					!node.statistics?.lwr,
```